src/features/auth/Unauthorised.tsx

```diff
@@ -19,28 +18,20 @@ export const Unauthorised: React.FC<Props> = () => {
   const [authState, send] = useActor(authService);
 
   if (authState.context.errorCode === "NO_WEB3") {
-    return (
-      <Panel>
-        <Web3Missing />
-      </Panel>
-    );
+    return <Web3Missing />;
   }
 
   if (authState.context.errorCode === "WRONG_CHAIN") {
-    return (
-      <Panel>
-        <WrongChain />
-      </Panel>
-    );
+    return <WrongChain />;
   }
 
   if (authState.context.errorCode === "NO_FARM") {
     return <Beta />;
   }
 
   return (
-    <Panel>
-      Catch all other connecting errors :/
+    <>
+      <span>Catch all other connecting errors :/</span>
```

Comment:
There are a few error states we can flesh out. We can also use the same error animation when a save fails :) 

Let's define some of these screens and we can tackle them in separate tickets 💪 @Rofrtd @spencerdezartsmith

Reply:
This this can easily be done @Rofrtd but would be good to knock out a bunch of error handling at once after working out a list. Definitely can create another ticket for this one.